### PR TITLE
Update index.json

### PR DESCRIPTION
**Pull request by tauridundee**

### CODE REVIEW

**Comment by tauridundee** [comment]
I accepted the invitation for write access but still can't seem to edit without a pull, maybe I'm doing this wrong?

**Comment by MikeMcQuaid** [comment]
@tauridundee You still haven't accepted your invitation, it seems.

File: _data/index.json [hunk 1]
```diff
@@ -42,6 +42,10 @@
       {
         "name": "Phil Leggetter",
         "talk": "Pusher and real-time messaging"
+      },
+      {
+        "name": "Chris Martin"
+        "talk": "A five finger introduction to software engineering"
```
Comment (MikeMcQuaid) [inline _data/index.json:48]: Sorry, should have checked this: there's a missing comma on the line above.

Comment (MikeMcQuaid) [inline _data/index.json:48]: Fixed in https://github.com/techmeetup/techmeetup.co.uk/commit/fa188f8ef761760abc0268a2a89446d8df43e3ff

**Comment by tauridundee** [comment]
I gave it a go but got hit with an error. I'll try it again. 

-Brian

**Pull request merged**